### PR TITLE
Code of conduct

### PR DESCRIPTION
**Pull request by crackofdusk**
Adds a new page with the code of conduct and links to it from the home page.

## Screenshots

Code of conduct:

Code of conduct page

Home page changes:

| Before | After |
|--|--|
| Home page with social media links in the hero section | Home page with social media and code of conduct links in the footer

### CODE REVIEW

**Comment by crackofdusk** [comment]
I continued with hand-written HTML, delaying the introduction of a more complicated build step until we need one.

I do think we *need* to do something about cache invalidation as we now have a new version of the stylesheet.

File: src/2023/code-of-conduct/index.html [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+            <li>Violence, threats of violence, or violent language.</li>
+            <li>Other disruptive or unsafe behavior.</li>
+          </ul>
+
+          <h2>Enforcement</h2>
+
+          <p>
+            If you are the subject of or witness to violations of this Code of Conduct,
+            please let us know in whatever way you feel comfortable: you can speak with us
+            directly or email <a href="email:conduct@helvetic-ruby.ch">conduct@helvetic-ruby.ch</a>. Organizers will take any action they deem
```
Comment (lxxxvi) [inline src/2023/code-of-conduct/index.html:68]: Isn't it `href="mailto: [...]"`?

**Review by lxxxvi — APPROVED** [review]
I like it, thanks for adding it!

Regarding the caching, for my website I have this in my GH pipeline.

https://github.com/schuettel-dev/website/blob/v3/bin/build#L19-L28

It hashes the CSS and replaces the path in the HTML files, may be we could do something similar?

File: src/2023/code-of-conduct/index.html (hunk 1 above)
Comment (crackofdusk) [reply src/2023/code-of-conduct/index.html:68]: Yes, I forgot to check when I added the link. 👍

**Comment by crackofdusk** [comment]
I'll merge https://github.com/MINASWAN/helvetic-ruby.ch/pull/8 first. Then I need to adjust `bin/build` again to account for the subdirectory created in the current pull request. (`sed` complains about `public/**/*.html` as a target if there are not subidrectories, like right now on the `main` branch)

**Review by hharen — APPROVED** [review]
Great, well done. 🚀

At some point we should make sure to go public as organizers and to be recognizable so that people actually can approach us.

**Pull request merged**